packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,183 @@
+
+interface OSCALPropertyProps {
+  /**
+   * Contains property elements
+   */
+  property: Property;
+}
+
+const OSCALProperty: React.FC<OSCALPropertyProps> = ({ property }) => {
+  const NO_INFORMATION = <NotSpecifiedTypography>Not Specified</NotSpecifiedTypography>;
+
+  return (
+    <StyledTooltip title={property.remarks ?? ""}>
+      <StyledTableRow key={property.uuid}>
+        <TableCell>{property.name ?? NO_INFORMATION}</TableCell>
+        <TableCell>{property.class ?? NO_INFORMATION}</TableCell>
+        <TableCell>{property.value ?? NO_INFORMATION}</TableCell>
+      </StyledTableRow>
+    </StyledTooltip>
+  );
+};
+
+interface OSCALPropertiesProps {
+  /**
+   * Contains a list of properties
+   */
+  properties?: Property[];
+  /**
+   * A namespace used for the title of a table
+   */
+  namespace?: string;
+}
+
+const OSCALProperties: React.FC<OSCALPropertiesProps> = ({ properties, namespace }) => {
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id={`${namespace}-table-title`}>
+        {namespace}
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer sx={{ maxHeight: "20em" }}>
+        <Table aria-label="Components" sx={{ height: "max-content" }}>
+          <StyledTableHead>
+            <TableRow>
+              <StyledHeaderTableCell>Name</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Class</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Value</StyledHeaderTableCell>
+            </TableRow>
+          </StyledTableHead>
+          <TableBody>
+            {properties
+              ?.filter((property) => namespaceOf(property?.ns) === namespace)
+              .map((property) => (
+                <OSCALProperty
+                  property={property}
+                  key={`${namespaceOf(property?.ns)}-properties`}
+                />
+              ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+};
+
+interface OSCALPropertiesDialogProps {
+  /**
+   * Contains properties elements
+   */
+  properties?: Property[];
+  /**
+   * Title of the element the properties resides under
+   */
+  title?: string | ReactElement;
+}
+
+export const OSCALPropertiesDialog: React.FC<OSCALPropertiesDialogProps> = ({
+  properties,
+  title,
+}) => {
+  const [open, setOpen] = React.useState(false);
+
+  if (!properties) {
+    return null;
+  }
+  const namespaceToProps: Record<string, Property[]> = {};
+  for (const prop of properties) {
+    const ns = namespaceOf(prop.ns);
+    namespaceToProps[ns] ??= [];
+    namespaceToProps[ns].push(prop);
+  }
+  const { [NIST_DEFAULT_NAMESPACE]: nist, ...thirdParties } = namespaceToProps;
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <OSCALPropertiesButton
+        size="small"
+        variant="outlined"
+        onClick={handleOpen}
+        aria-label={`${title} properties button`}
```

Comment:
`title` may not be a string which won't work well in an `aria-label`. Also, I am not sure that "button" is appropriate here... We may want to use something like the other suggestion for the `title` attribute.

Reply:
I could just make this "open-properties" or should I be more specific? I was using `title` to differentiate them a little more, but I understand why that wouldn't work well.